Assets/MRTK/Services/InputSimulation/SimulatedGestureHand.cs

```diff
@@ -98,14 +103,15 @@ private void EnsureProfileSettings()
             }
         }
 
+
         /// <summary>
         /// The GGV default interactions.
         /// </summary>
         /// <remarks>A single interaction mapping works for both left and right controllers.</remarks>
         public override MixedRealityInteractionMapping[] DefaultInteractions => new[]
         {
-            new MixedRealityInteractionMapping(0, "Select", AxisType.Digital, DeviceInputType.Select),
-            new MixedRealityInteractionMapping(1, "Grip Pose", AxisType.SixDof, DeviceInputType.SpatialGrip),
+            new MixedRealityInteractionMapping(0, "Select", AxisType.Digital, DeviceInputType.Select, selectAction),
```

Comment:
Actions here should be obtained from the controller mapping profile. We shouldn't define actions for these mappings in code like this. If you're seeing a case where mappings are being obtained without the actions expected from the profile, we should figure out the issue there (serialization maybe, if you're adding a new handedness option).